src/main/scala/org/apache/pekko/persistence/dynamodb/journal/DynamoDBJournal.scala

```diff
@@ -117,7 +115,7 @@ class DynamoDBJournal(config: Config)
     f.onComplete { _ =>
       self ! OpFinished(pid, p.future)
       p.success(Done)
-    }(akka.dispatch.ExecutionContexts.sameThreadExecutionContext)
+    }(org.apache.pekko.dispatch.ExecutionContexts.sameThreadExecutionContext)
```

Comment:
It would be idea that for these cases like this we have `pekko.dispatch.ExecutionContexts.sameThreadExecutionContext` rather than `org.apache.pekko.dispatch.ExecutionContexts.sameThreadExecutionContext`

Reply:
I'll fix this one up - but generally, I am not sold on the `import org.apache.pekko` solution. Are we even sure all the tools we use support it (scalafix, scalafmt, whatever gives us the `sortImports` task)?